prime-dotnet-webapi/ViewModels/Business Events/_BusinessEventMappingProfile.cs

```diff
@@ -0,0 +1,15 @@
+using AutoMapper;
+using Prime.Models;
+
+namespace Prime.ViewModels.Profiles
+{
+    public class BusinessEventMappingProfile : Profile
+    {
+        public BusinessEventMappingProfile()
+        {
+            CreateMap<BusinessEvent, SiteBusinessEventViewModel>()
+                .ForMember(dest => dest.PartyName, opt => opt.MapFrom(src => src.Party.FirstName + ' ' + src.Party.LastName))
+                .ForMember(dest => dest.AdminIDIR, opt => opt.MapFrom(src => src.AdminIDIR));
```

Comment:
`AdminIDIR` is not a `[Cumputed]` property on the `BusinessEvent` model; so while this mapping may (or may not!) succeed, the generated SQL will probably be rather weird looking. Likely, it will still fetch the entire `Admin` just to discard everything but the IDIR. you could map the property as `[Computed]` and ass `.DecompileAsync()`, or just do
```C#
CreateMap<BusinessEvent, SiteBusinessEventViewModel>()
    .ForMember(dest => dest.PartyName, opt => opt.MapFrom(src => src.Party.FirstName + ' ' + src.Party.LastName))
    .ForMember(dest => dest.AdminIDIR, opt => opt.MapFrom(src => src.Admin.IDIR));
```

Reply:
As an aside, AutoMapper would do this mapping for us automatically if we *didn't* have `AdminIDIR` on the `BusinessEvent` model. The first thing checks for is matching names, but AutoMapper will _also_ use naming conventions and will map from `Source.Child.Property` to `Destination.ChildProperty`